app/controllers/items_controller.rb

```diff
@@ -5,18 +5,34 @@ class ItemsController < ApplicationController
   before_action :find_item, only: %i[show toggle_favorite]
 
   def index
-    # Filter by postal code
-    if params[:postal_code].present?
+    # Gets current user coordinates, currently using static postal code until we fix the Js issue.
+    @current_postal_code = 'H2T 1X3'
+
+    # Distance between current user and other users (for index cards).
+    @distances_between_other_users = {}
+
+    # Sets distance for each user that's nearby, private method below.
+    set_distance
+
+    if @current_postal_code.present?
       # Filter users if items are near (5km)
-      @users = User.near(params[:postal_code], 5)
+      @users = User.near(@current_postal_code, 5)
+
       # Get all of these users items
       @items = @users.map {|u| u.items}.flatten
+
+      # This condition is nested so that the cards will still have the distance from the user.
+      if params[:query].present?
```

Comment:
This isn't duplicate, one condition checks for a search with a postal code present, the other just checks for a search.